foundation-extensions/mobile-core/configuration/README.md

```diff
@@ -263,6 +263,48 @@ ACPCore.UpdateConfiguration(config);
 {% endtab %}
 {% endtabs %}
 
+## Clearing programmatic updates to configuration (iOS 3.x.x Only)
+
+You can clear any programmatic updates made to configuration via the `clearUpdatedConfiguration` API. This will clear programmatic updates to configuration made via the `updateConfigurationWith(configDict:)` API. It will also clear any updates to the `PrivacyStatus` made via `setPrivacyStatus(_ status:)`. 
+
+Here are some examples of scenarios:
+
+`configureWith(appId:)` -> `updateConfigurationWith(configDict:)` -> `clearUpdatedConfiguration()` = You end up with the initial configuration set via `configureWith(appId:)`
```

Comment:
Not a huge fan of using equal signs to show what it means.

I'd write it out in sentence form instead, so something like - "... -> `clearUpdatedConfiguration()`: In this example, you end up with...".